#goal definition
trajectory_custom_msgs/PointArray waypoints
---
#result definition
bool success
---
#feedback
float64 percentage
geometry_msgs/PoseArray ee_pose